### PR TITLE
Fix keyboard interactive

### PR DESCRIPTION
**Pull request by neatdecisions**
1) helpers.js: fix a condition in normalizeConfig which was always setting user-defined onKeyboardInteractive callback to null.
2) README.md: fix an example.

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -89,10 +89,12 @@ export async function normalizeConfig(givenConfig: ConfigGiven): Promise<Config>
 
   config.tryKeyboard = !!config.tryKeyboard
 
-  if (config.tryKeyboard === true && typeof config.onKeyboardInteractive !== 'function') {
-    config.onKeyboardInteractive = (name, instructions, instructionsLang, prompts, finish) => {
-      if (prompts.length > 0 && prompts[0].prompt.toLowerCase().includes('password')) {
-        finish([config.password])
+  if (config.tryKeyboard === true) {
```
Comment (steelbrain) [inline src/helpers.js]: I think we could word the check differently to make it
```
if (typeof config.onKeyboardInteractive !== 'function' || config.tryKeyboard)
```

That should satisfy the cases we want

**Comment by steelbrain** [comment]
CI is failing on prettier, could you please prettify the changed files? Cheers

**Comment by neatdecisions** [comment]
Done, thanks!

**Comment by steelbrain** [comment]
Thanks again!

**Comment by steelbrain** [comment]
Fix published in v5.1.1

**Pull request merged**